.appveyor.yml

```diff
@@ -43,11 +43,12 @@ build_script:
           -DCMAKE_TOOLCHAIN_FILE=%VCPKG_DIR%\scripts\buildsystems\vcpkg.cmake
           -DPCL_BUILD_WITH_BOOST_DYNAMIC_LINKING_WIN32=ON
           -DPCL_BUILD_WITH_FLANN_DYNAMIC_LINKING_WIN32=ON
-          -DPCL_NO_PRECOMPILE=ON
+          -DPCL_NO_PRECOMPILE=OFF
           -DBUILD_simulation=ON
-          -DBUILD_global_tests=OFF
+          -DBUILD_global_tests=ON
           -DBUILD_examples=OFF
           -DBUILD_tools=OFF
           -DBUILD_apps=OFF
           ..
   - cmake --build . --config %CONFIGURATION%
+  - ctest -C %CONFIGURATION% -V
```

Comment:
Don't forget to add this new line later.

Reply:
Fixed